apps/style/netsim/style.scss

```diff
@@ -1,7 +1,15 @@
 $root: '/blockly/media/turtle/'; //TODO: Parameterize for asset pipeline
 
+$white: #ffffff;
+$whitesmoke: #f5f5f5;
+$medium_gray: #ddd;
+$dark_gray: #A9A9A9;
```

Comment:
Color `#A9A9A9` should be written as `#a9a9a9`<br>Name of variable `dark_gray` should be written in all lowercase letters with hyphens instead of underscores